README.md

```diff
@@ -30,6 +30,10 @@ We have a discord where we discuss about out updates, features, requests, best p
 
 [Join the Envless Discord](https://discord.gg/XEDCq7nS)
 
+## 🛡️ License
+
+LinkFree is licensed under the GNU Affero General Public License v3.0 License - see the [LICENSE](https://github.com/envless/envless/blob/main/LICENSE) file for details.
```

Comment:
Please update this with Envless and proper license
https://github.com/envless/envless/blob/main/LICENSE
🤔